onnxruntime/core/providers/coreml/builders/impl/activation_op_builder.cc

```diff
@@ -54,6 +110,68 @@ Status ActivationOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_builder,
 
 // Operator support related
 
+namespace {
+// assumes that node.OpType() == "PRelu"
+bool IsPReluOpSupported(const Node& node, const OpBuilderInputParams& input_params,
+                        const logging::Logger& logger) {
+  const auto& input_defs = node.InputDefs();
+
+  // X input rank must be 3 or 4
+  std::vector<int64_t> x_shape;
+  if (!GetShape(*input_defs[0], x_shape, logger)) {
+    return false;
+  }
+
+  const auto x_rank = x_shape.size();
+  if (x_rank == 3 || x_rank == 4) {
```

Comment:
actually just tried to run perf_test with superRes model, the PRelu op still shows not supported and returned on this line.

Wondering should this be ``` !(x_rank == 3 || x_rank == 4) ``` instead?

Reply:
oops, sorry about that. yes, will create another PR